### PR TITLE
Optimize capture verification method in China region

### PR DESCRIPTION
**Pull request by Yixi**
## Proposed change

Optimized the logic of automatically passing the graphical captcha during login in the China region.

## Type of change
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to this library)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

Recently found that it is not possible to obtain the verification code through polling. As long as the slider capture has more than two or three incorrect attempts, the current slider capture will become unavailable. Introduced the Pillow library to automatically identify the position value required for the slider capture image processing. Currently, it can basically ensure that it can directly log in every time.
There will no longer be multiple requests errors when attempting to use a verification code.

image

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 

## Checklist

- [X] The code change is tested and works locally.
- [ ] Tests have been added to verify that the new code works.

### CODE REVIEW

**Comment by Yixi** [comment]
What's missing before the merge? @rikroe

**Comment by rikroe** [comment]
Just move `types-Pillow` to `requirements-test.txt` (as mentioned in the review) and all should be good.

**Comment by Yixi** [comment]
> Just move `types-Pillow` to `requirements-test.txt` (as mentioned in the review) and all should be good.

Hmm.. Maybe I missed the review message, but I've already move `types-Pillow` to `requirements-test.txt`

File: requirements.txt
Comment (rikroe) [inline requirements.txt:4]: Is there a reason for version pinning?

**Review by rikroe — CHANGES_REQUESTED** [review]
Just realized that `Pillow` should be added to `setup.py` as well.

File: requirements.txt
Comment (Yixi) [reply requirements.txt:4]: Just import the latest version

Comment (rikroe) [reply requirements.txt:4]: In that case, please remove the `>=6.2.0`.
The HA maintainers do have a look on these if we update the HA component and prefer as little package constraints as possible.

**Review by rikroe — APPROVED** [review]
LGTM, thank you!

**Pull request merged**
